features/clipboard-unsanitized-formats.yml

```
@@ -0,0 +1,10 @@
+name: Paste unsanitized data
+description: The optional `formats` parameter of the `Clipboard.read()` method pastes unsanitized data from the system clipboard.
```

`Clipboard.read()` isn't a static method, but we could do this in this case:

```suggestion
description: The optional `formats` parameter of the `navigator.clipboard.read()` method reads unsanitized data from the system clipboard.
```

(Also "reads" instead of "pastes".)

@ddbeck what style do you think we should use for a parameter of a method of an interface? It's not the parameter that does the reading, it's still the method, but how to phrase that?

---

Hmm, wait, am I wrong about paste vs. read? From the user point of view you copy and paste. As the user of the clipboard API you read from or write to the clipboard. Write to copy. Read to paste. Right?

I have no view on what terminology is easier to understand.

---

Correct: paste == read.